sdk/component/internal/storer/local/sqlite/sqlite_test.go

```diff
@@ -0,0 +1,218 @@
+package sqlite_test
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/smithy-security/smithy/sdk/component"
+	"github.com/smithy-security/smithy/sdk/component/internal/storer"
+	"github.com/smithy-security/smithy/sdk/component/internal/storer/local/sqlite"
+	"github.com/smithy-security/smithy/sdk/component/internal/uuid"
+	ocsf "github.com/smithy-security/smithy/sdk/gen/com/github/ocsf/ocsf_schema/v1"
+)
+
+const (
+	dbName = "smithy.db"
+)
+
+type (
+	localStorer interface {
+		component.Closer
+		component.Reader
+		component.Updater
+		component.Writer
+
+		CreateTable() error
+	}
+
+	ManagerTestSuite struct {
+		suite.Suite
```

Comment:
Using https://pkg.go.dev/github.com/stretchr/testify/suite for nice bdd like setup